src/components/oculus-touch-controls.js

```diff
@@ -234,7 +234,9 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       var value = axesMapping[key];
       var detail = {};
       var changed = !evt.detail.changed;
-      if (!changed) { value.forEach(function (axisNumber) { changed |= evt.detail.changed[axisNumber]; }); }
+      if (!changed) {
```

Comment:
it has the side effect of making more garbage in detail even if not changed, which is less desirable